.pa11yci

```diff
@@ -12,6 +12,6 @@
       "htmlcs",
       "axe"
     ],
-    "hideElements": ".usa-select, .usa-step-indicator__segment-label"
+    "hideElements": ".usa-select, .usa-step-indicator__segment-label, .is-hidden"
```

Comment:
`pa11y` was giving a false positive on the empty modal shell even though it's not in the accessibility tree.